pandexo/engine/run_online.py

```diff
@@ -589,7 +589,7 @@ def post(self):
                     inc = planet_data['inclination']
 
                 exodata["planet"]["i"]          = inc
-                exodata["planet"]["ars"]        = planet_data['a/Rs'] 
+                exodata["planet"]["a"]        = planet_data['a/Rs'] 
```

Comment:
In general, changing the dictionary keys of exo dict or the inst dict will mess up the people using the Python API. 

https://natashabatalha.github.io/PandExo/tutorialhst.html#edit-stellar-and-planet-inputs

Reply:
I see, I can make the change.

Reply:
Awesome thanks! Great work on this!!